src/lib.rs

```diff
@@ -7,7 +7,7 @@
 mod alignment;
 mod archive;
 mod archive_writer;
-mod coff;
+pub(crate) mod coff;
```

Comment:
You don't need `pub(crate)` here. Items in the crate root are always effectively `pub(crate)`.